newIDE/app/src/AssetStore/InstallAsset.js

```diff
@@ -119,6 +119,8 @@ export const installResource = (
     newResource = new gd.VideoResource();
   } else if (serializedResource.kind === 'json') {
     newResource = new gd.JsonResource();
+  } else if (serializedResource.kind === 'model3D') {
```

Comment:
same here, should it also handle cube3D in case we put some in the asset store in the future? I think it will happen at some point if we embed an asset pack for kenney 3D platformer for instance